recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
```

Comment:
This line should go. It is saver to do that in the `.CI` (and it was already done.)